test/test_circle_row.ml

```diff
@@ -0,0 +1,22 @@
+open Joy.Shape
+
+let row_of_circles dim_x dim_y spacing =
+  let circles = ref [] in
+  let total_width = 3 * spacing in 
+  let (center_x, center_y) = (dim_x / 2, dim_y / 2) in
+  let start_x = center_x - total_width / 2 in 
+  for i = 0 to 2 do
+    let x = start_x + i * spacing in
+    circles := circle ~x:x ~y:center_y 50 :: !circles
+  done;
+  !circles
+
+let () =
+  let (dim_x, dim_y) = (1000,1000) in
+  set_dimensions dim_x dim_y;
+  init ();
+  let spacing = 120 in 
+  let circles = row_of_circles dim_x dim_y spacing in
+  show circles;
+  ignore (read_line ());
+  close()
```

Comment:
@kushalpokharel this example can be made simpler with the recent `translate` transformation that was added. Can you please use that instead?

Here is an example of its usage -- https://github.com/Sudha247/ocaml-joy/blob/main/examples/translate_circle.ml

Some other pointers:
- please stick with the default for dimensions
- i don't think we need another function, or to use variables
- please don't use references (like `circles` in this example)

as much as possible, we want our examples to be inviting to newbies. simplicity will go a long way in that!